x-pack/plugins/security_solution/public/management/pages/event_filters/view/event_filters_list_page.tsx

```diff
@@ -126,9 +129,15 @@ export const EventFiltersListPage = memo(() => {
     [navigateCallback]
   );
 
-  const handleItemDelete: ExceptionItemProps['onDeleteException'] = useCallback((args) => {
-    // TODO: implement delete item
-  }, []);
+  const handleItemDelete: ExceptionItemProps['onDeleteException'] = useCallback(
+    ({ id }) => {
+      dispatch({
+        type: 'eventFilterForDeletion',
+        payload: listItems.find((item) => item.id === id)!,
```

Comment:
About the Pick -> pick(listItems.find((item) => item.id === id)!, ['id', 'name'])

Reply:
Yeah, I guess I could use lodash `pick`. But now I'm thinking that perhaps I just do a cast here and use the entire record.

Reply:
fixed. added a cast to remove `Immutable<>` from item returned